backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/CassandraCluster.java

```diff
@@ -80,7 +80,7 @@ public void clearAllTables() {
     private Optional<Session> tryInitializeSession() {
         try {
             Cluster clusterWithInitializedKeyspace = ClusterWithKeyspaceCreatedFactory
-                .clusterWithInitializedKeyspace(getCluster(), KEYSPACE_NAME, REPLICATION_FACTOR);
+                .clusterWithInitializedKeyspaceWithoutDurableWrites(getCluster(), KEYSPACE_NAME, REPLICATION_FACTOR);
```

Comment:
Really, really cool ! 
However, I wonder if it gets apply with the EmbbeddedCassandra during jmap-integration-testing. Could you check ?